src/main/java/uk/gov/pay/api/model/telephone/CreateTelephonePaymentRequest.java

```diff
@@ -154,4 +175,122 @@ public String getFirstSixDigits() {
     public String getTelephoneNumber() {
         return telephoneNumber;
     }
+
+    public static class TelephoneRequestBuilder {
+        private int amount;
+
+        private String reference;
+
+        private String description;
+
+        private String createdDate;
+
+        private String authorisedDate;
+
+        private String processorId;
+
+        private String providerId;
+
+        private String authCode;
+
+        private PaymentOutcome paymentOutcome;
+
+        private String cardType;
+
+        private String nameOnCard;
+
+        private String emailAddress;
+
+        private String cardExpiry;
+
+        private String lastFourDigits;
+
+        private String firstSixDigits;
+
+        private String telephoneNumber;
+
+        public TelephoneRequestBuilder amount(int amount) {
```

Comment:
in other places the pattern is `with<field>` .ie `withAmount`.

Reply:
Changed.